wolfssl/internal.h

```diff
@@ -2201,13 +2201,16 @@ WOLFSSL_LOCAL int DoVerifyCallback(WOLFSSL_CERT_MANAGER* cm, WOLFSSL* ssl,
 /* wolfSSL Sock Addr */
 struct WOLFSSL_SOCKADDR {
     unsigned int sz; /* sockaddr size */
+    unsigned int bufSz; /* sockaddr size */
     void*        sa; /* pointer to the sockaddr_in or sockaddr_in6 */
 };
 
 typedef struct WOLFSSL_DTLS_CTX {
     WOLFSSL_SOCKADDR peer;
     int rfd;
     int wfd;
+    byte userSet:1;
+    byte connected:1; /* Set when the rfd and wfd are connected sockets */
```

Comment:
Can we come up with a better name for this member? It seems to only change the use of the "peer" sockaddr in the recvfrom and sendto.

Reply:
I honestly can't think of a better name that won't turn into a Java-esque sentence. When `connected` is set, this indicates that the sockets in `rfd` and `wfd` are "connected" (`connect()` was called on them in addition to `bind`). This means that `sendto` and `recvfrom` don't need to specify and store the peer address. Also if I remember correctly, @ejohnstown reported on another PR that using `sendto` on a connected socket on a Mac resulted on a failure. This may require us to switch to using just `send` in `EmbedSendTo` on a connected socket.

In short: can you think of a better name to describe this behaviour because I can't?

Reply:
maybe `connectedSocket` or `connectedDatagramSocket`

Reply:
I don't think that is a better name. Its too verbose and `connected` in the context of the `WOLFSSL_DTLS_CTX` object already only applies to the socket.